main.js

```diff
@@ -519,8 +519,13 @@ function generateFilename(prefix, suffix) {
     var date = new Date();
     var filename = prefix;
 
-    if (CONFIG && CONFIG.name && CONFIG.name.trim() !== '') {
-        filename = filename + '_' + CONFIG.name.trim().replace(' ', '_');
+    if (CONFIG) {
+        if (CONFIG.flightControllerIdentifier && CONFIG.flightControllerIdentifier.trim() !== '') {
```

Comment:
As far as I know I've seen that the `flightControllerIdentifier` is the first thing readed after verify if the API is supported, so you're right, it will never must be empty.
So I have modified the PR.